src/hooks/screen/useChatScreenHook.ts

```diff
@@ -6,6 +6,7 @@ import _ from 'lodash';
 import SimpleToast from 'react-native-simple-toast';
 import {v4 as uuid} from 'uuid';
 
+import {useMutation} from 'react-query';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useMutation` from 'react-query' is added. This is a part of the react-query library that allows you to use mutations (data changing operations) in your components.
```diff
+ import {useMutation} from 'react-query';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->